desc/objectives/_coils.py

```diff
@@ -903,6 +910,11 @@ class PlasmaCoilSetMinDistance(_Objective):
         a large number of coils, or if the resolution is very high, setting this to a
         small value will reduce peak memory usage at the cost of slightly increased
         runtime.
+    use_centers : bool
+        Whether or not to only consider the coil centers, rather than the position
+        along the coil itself. This can reduce the computational cost in situations
+        where a coil's position is well-approximated by its center, for example
+        if the coils have a very small radius.
```

Comment:
see above comment, and should also mention that the coils shouldn't link the plasma (is it possible to also check that the initial coils dont link? not sure how expensive that is)

Reply:
Might be a bit expensive, since users have to opt into this flag I think I will trust them to know when they can't use it.